pages/_app.js

```diff
@@ -2,10 +2,18 @@ import GlobalStyle from "../styles";
 import Head from "next/head";
 import { useState } from "react";
 import NavBar from "../components/NavBar/NavBar";
+import { initalRides } from "../lib/rides";
 
 export default function App({ Component, pageProps }) {
+  const [rides, setRides] = useState(initalRides);
   const [bookedRides, setBookedRides] = useState([]);
-  console.log(bookedRides);
+  console.log("Rides in der App.js:", rides);
```

Comment:
Try to avoid merging `console.log` into main
```suggestion
  console.log("Rides in der App.js:", rides);
```